views/service.html

```diff
@@ -9,12 +9,12 @@
     <dd>{{resource.metadata.creationTimestamp}}</dd>
     <dt>IP</dt>
     <dd>{{resource.spec.portalIP}}</dd>
-    <dt>Port</dt>
-    <dd>{{resource.spec.port}}</dd>    
-    <dt>Container port</dt>
-    <dd>{{resource.spec.containerPort}}</dd>
-    <dt>Protocol</dt>
-    <dd>{{resource.spec.protocol}}</dd>
+    <dt>Port(s)</dt>
+    <dd>
+      <div ng-repeat="portMapping in resource.spec.ports | orderBy:port">
```

Comment:
Not sure if a `<div>` inside a `<dd>` will mess up layout... I think we're trying to get things side-by-side if the rendering area is wide enough. Take a look at how containers.html renders multiple ports... not sure I'm a fan of it (empty `<dt>` for all ports but the first), but at least it keeps things side-by-side